addons/pos_self_order/controllers/qr_menu.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+
+from typing import Dict, Optional
+import werkzeug
+
+from odoo import http
+from odoo.http import request
+
+from odoo.addons.pos_self_order.controllers.utils import (
+    get_pos_config_sudo,
+    get_any_pos_config_sudo,
+    get_table_sudo,
+)
+from odoo.addons.pos_self_order.models.pos_config import PosConfig
+from odoo.addons.pos_self_order.models.pos_restaurant import RestaurantTable
+
+
+class PosQRMenuController(http.Controller):
+    """
+    This is the controller for the POS Self Order App
+    There is one main route that the client will use to access the POS Self Order App: /menu
+    This route will render the LANDING PAGE of the POS Self Order App
+    And it will pass the needed data to the template: the list of products, pos_config_id, table_id, company_name, currency...
+    After that the client will be able to navigate to the /products route w/o aditional requests
+    to the server, using client side routing.
+    """
+
+    @http.route("/menu", auth="public")
+    def pos_self_order_redirect(self):
+        return request.redirect(f"/menu/{get_any_pos_config_sudo().id}")
```

Comment:
get any pos? What if there is no active session in that config? I guess we can show the menu but the ordering feature should not be active.